packages/stripe_platform_interface/lib/src/method_channel_stripe.dart

```diff
@@ -424,10 +424,20 @@ class MethodChannelStripe extends StripePlatform {
   }) async {
     bool? isSupported;
     if (params == null) {
-      isSupported =
-          await _methodChannel.invokeMethod('isPlatformPaySupported', {
+      final result =
+          await _methodChannel.invokeMethod<dynamic>('isPlatformPaySupported', {
         'params': {},
       });
+
+      if (result is bool) {
+        isSupported = result;
+      } else {
+        throw StripeException(
```

Comment:
That would then result in a different exception rather than a fix. Did you find out what the result contains if not bool?

Reply:
sorry forgot to force push 😂